src/pages/PageMain/index.tsx

```diff
@@ -119,7 +127,15 @@ export const CardToken: React.FC<TypeCardProps> = ({ token, to = '/' }) => {
 export const PageMain: React.FC = () => {
   const { tokens } = useSelector(({ zx }: any) => zx);
 
-  const [tokensList, setTokensList] = React.useState<TypeToken[]>([]);
+  const [tokensList, setTokensList] = React.useState<TypeToken[]>(
+    new Array(4).fill('').map(() => {
+      return {
+        symbol: '',
+        name: '',
+        address: '',
+      };
+    }),
```

Comment:
попробуй сократить код до такого вида:

```suggestion
    new Array(4).fill('').map(() => ({symbol: '',name: '',address: ''})),
```

Reply:
Fixed